client/actions/bookActions.js

```diff
@@ -0,0 +1,70 @@
+import actionTypes from './actionTypes';
```

Comment:
I can see some es5 syntax in this file e.g using the function keyword. Make sure to correct accordingly.

Reply:
I'm going to remove the contents, I'm not actually using that anymore. Thanks